Include/Rover/Model.hpp

```diff
@@ -82,15 +83,15 @@ namespace Rover {
       auto operator ()(const Arguments& args) const;
 
     private:
-      using ScalarSample = ScalarSample<ComputationType>;
```

Comment:
Go back to the name ```ScalarSample```. To disambiguate, do something like:

    using ScalarSample = Rover::ScalarSample<ComputationType>;